go/interpreter/lfvm/interpreter_test.go

```diff
@@ -117,7 +117,7 @@ type example struct {
 	function uint32 // The identifier of the function in the contract to be called
 }
 
-const MAX_STACK_SIZE int = 1024
+const MAX_STACK_SIZE int = 1024 // TODO: use StackLimit
```

Comment:
stack limit is now public, did you miss it, or can't it be used yet?

Reply:
for some reason I could not use this constant, this is why I made `StackLimit` public.

Reply:
I don't understand your reply.
- does it mean that it does not need to be public anymore? 
- it can be used, I tested it. Even better, we can remove MAX_STACK_SIZE and use StackLimit

Reply:
I tried using this constant from ct package, but compiler complained it did not exist. I would prefer to have just one constant rather than 2.